--- spambayes-1.0a2/pspam/zeo.sh.orig	Mon Nov  4 05:44:20 2002
+++ spambayes-1.0a2/pspam/zeo.sh	Fri Feb 14 00:10:34 2003
@@ -1,5 +1,7 @@
-#! /bin/bash
+#! /bin/sh
 
-export STUPID_LOG_FILE=/var/tmp/zeospam.log
-export LIBDIR=/usr/local/lib/python2.3/site-packages
-python2.3 $LIBDIR/ZEO/start.py -U /var/tmp/zeospam /var/tmp/zeospam.fs
+STUPID_LOG_FILE=/var/tmp/zeospam.log
+export STUPID_LOG_FILE
+LIBDIR=/usr/lib/python2.3/site-packages
+export LIBDIR
+python $LIBDIR/ZEO/start.py -U /var/tmp/zeospam /var/tmp/zeospam.fs
--- spambayes-1.0a2/contrib/bulktrain.sh.orig	Thu Jan 23 05:46:28 2003
+++ spambayes-1.0a2/contrib/bulktrain.sh	Fri Feb 14 00:14:08 2003
@@ -1,7 +1,7 @@
-#!/bin/bash
+#!/bin/sh
 cd $HOME/spambayes/active/spambayes
 rm -f tmpdb 2>/dev/null
-time /usr/bin/python2.2 bulkgraph.py \
+time /usr/bin/python bulkgraph.py \
  -d tmpdb \
  -e $HOME/Mail/everything/ \
  -s $HOME/Mail/spam \
